pkg/apiserver/experiment/experiment.go

```diff
@@ -63,3 +68,192 @@ func (s *Service) getExperimentDetail(c *gin.Context) {}
 
 // TODO: need to be implemented
 func (s *Service) deleteExperiment(c *gin.Context) {}
+
+// getPodChaosState returns the state of PodChaos
+func (s *Service) getPodChaosState (stateInfo map[string]int) error {
+	var chaosList v1alpha1.PodChaosList
+	err := s.kubeCli.List(context.Background(), &chaosList)
+	if err != nil {
+		return err
+	}
+	for _, chaos := range chaosList.Items {
+		stateInfo["Total"]++
+		switch chaos.Status.ChaosStatus.Experiment.Phase {
+		case v1alpha1.ExperimentPhaseRunning:
+			stateInfo["running"]++
+		case v1alpha1.ExperimentPhasePaused:
+			stateInfo["Paused"]++
+		case v1alpha1.ExperimentPhaseFailed:
+			stateInfo["Failed"]++
+		case v1alpha1.ExperimentPhaseFinished:
+			stateInfo["Finished"]++
+		}
```

Comment:
This switch statement is not needed I guess.
We can simply use 

``` go
for _, chaos := range chaosList.Items {
  stateInfo[string(chaos.Status.ChaosStatus.Experiment.Phase)]++
}
stateInfo["Total"] += len(chaosList.Items)
```

Reply:
Ok, I will change it.